bin/cml-runner.js

```diff
@@ -256,11 +267,29 @@ const run = async (opts) => {
     await tf.save_tfstate({ tfstate, path });
   }
 
+  // if (name !== NAME) {
+  //   console.log(
+  //     'The --name option is deprecated: please use --labels to differentiate runners instead.'
+  //   );
+  // }
+
   await cml.repo_token_check();
-  if (await cml.runner_by_name({ name }))
-    throw new Error(
-      `Runner name ${name} is already in use. Please change the name or terminate the other runner.`
-    );
+
+  if (reuse_existing && (await cml.runners_by_labels({ labels })).length > 0) {
```

Comment:
The `.length > 0` unnecessary comparison was just to exemplify how to keep a minimum number fo runners alive, but can be removed for the sake of cleanliness.

Reply:
```suggestion
  if (reuse_existing && (await cml.runners_by_labels({ labels }))) {
```